packages/app/app/actions/playlists.js

```diff
@@ -70,3 +68,47 @@ export function updatePlaylist(playlist) {
     });
   };
 }
+
+export function addPlaylistFromFile(filePath) {
+  return async dispatch => {
+    fs.readFile(filePath, (err, data) => {
+      if (err) {
+        dispatch(error('Playlist import fail', 'Can not open file', null, null));
```

Comment:
Just remembered this: these message strings need to be localized and defined in `en.json`

Reply:
I'll try to pass `t` from `useTranslation` from the container

Reply:
Ok, that's the best way to do it. Or just pass an object with message strings as an argument.